desktop-environment.el

```diff
@@ -357,14 +357,14 @@ replacing the placeholder %d with the prefix argument."
 (defun desktop-environment-toggle-mute ()
   "Toggle between muted and un-muted."
   (interactive)
-  (message "%s"
+  (message "Volume %s"
```

Comment:
this result will be bad for users who have configured `desktop-environment-volume-toggle-command` to use a different tool. Can you please change the default value of the variable to echo "Volume" instead?